app/mailers/user_mailer.rb

```diff
@@ -242,13 +242,15 @@ def add_email_associated_with_another_account
     end
   end
 
-  def account_verified(date_time:, sp_name:)
+  def account_verified(date_time:, profile:)
+    attachments.inline['verified.png'] = File.read('app/assets/images/email/user-signup-ial2.png')
```

Comment:
Remind me to complain some day that the `attach_images` before_action here doesn't do what the name implies, and only attaches the Login.gov logo.

Reply:
1. you could rename to `attach_logo` pretty easily I bet!
2. maybe specify `Rails.root` in there?

```suggestion
    attachments.inline['verified.png'] = Rails.root.join('app/assets/images/email/user-signup-ial2.png').read
```

Reply:
Will change to use `Rails.root`. I modeled my implementation on what `attach_images` does, but we've already established I'm not its biggest fan. ;)

I was, in fact, tempted to rename it to `attach_logo`, but I'd actually rather make it do what its name suggests it does, or at least make it easier to attach images, and that's too far out of scope of this story.